_component_design/key-value-pairs.md

```diff
@@ -1,40 +1,120 @@
+
+{% endcolumn %}
+</div>
+
+## Specifications and states
 
 <div class="hxRow">
-{% column left:"hxCol-4" %}
--   Labels should correspond with the labels used for data creation and/or
-  editing.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+Key value pairs include the following specifications and states:
+
+### Default layout
+
+Key value pairs are contained in the body of the page by default. Key values (i.e. "Ticket ID") are displayed on the left and aligned right. Data values (i.e. "Summary") are displayed on the right and are aligned left.
 
--   Labels appear to the left and are right aligned.
+To determine the indentation of key value pairs, use 2 columns on the Helix grid for key values and 4 columns for data values. Space between the key and data values must align with the grid gutter between the 2nd and 3rd columns.
 
--   Data values are displayed to the right and left aligned.
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"This example uses a spacing guideline for a 1200px screen size. Spacing widths might vary depending on your screen size"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/key-pairs/keypairs-default-layout.svg)
 
--   Icons should be used sparingly.
+{% endfigure %}
 {% endcolumn %}
-{% column right:"hxCol-8" %}
-{% figure [caption:"Example of key values pattern displaying server details"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/key-value-pairs.svg){:width="100%"}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Two lines
+
+In certain cases, key value pair text can wrap to two lines. We recommend formatting key value's across two columns and data value's across 4.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"A set of key value pairs that use two lines"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/key-pairs/keypairs-two-lines.svg)
+
 {% endfigure %}
 {% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Other elements
+
+Icons, compact tertiary buttons, and text links can be used in key value pairs.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Key value pairs that contain icons, buttons, and text links"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/key-pairs/keypairs-other-elements.svg)
+
+{% endfigure %}
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Loading state
+
+If a data value are still being called, list the key value, and then the word "Loading" followed by an ellipses for the data value.
```

Comment:
Use singular for the verb in the first clause if you're going to use `a data value`. Otherwise, remove `a` and change to `values`.